boards/arm/actinius_icarus/actinius_icarus_common.dts

```diff
@@ -86,7 +86,7 @@
 	};
 
 	sim_select: sim-select {
-		compatible = "sim-select";
+		compatible = "actinius,sim-select";
```

Comment:
I was precisely told not to do this: https://github.com/zephyrproject-rtos/zephyr/pull/43817#discussion_r831620227 cc: @mbolivar-nordic

Reply:
@mbolivar-nordic ping

Reply:
@mbolivar-nordic @galak I can change it back, but can we please make sure there is a consensus first?

Reply:
Sounds good -- @galak, you OK with removing the invalid vendor prefix?

Reply:
@galak ping

Reply:
@galak ping

Reply:
@galak @gmarull @mbolivar-nordic could you please let me know whether it should be with or without the vendor prefix so that we get these fixes in as soon as possible? The board files are broken as they are in zephyr right now.